src/app/kbv/controllers/question.test.js

```diff
@@ -81,6 +81,65 @@ describe("Question controller", () => {
     });
   });
 
+  describe("configured question undefined", () => {
+    let prototypeSpy;
+    let questionToTranslationsSpy;
+
+    beforeEach(() => {
+      req.form.options = {
+        fields: {},
+      };
+
+      req.session.question = undefined;
```

Comment:
FYI, another way of doing this is to use the [delete operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/delete)